analyzer/src/funTest/kotlin/managers/SpdxDocumentFileFunTest.kt

```diff
@@ -54,7 +54,26 @@ class SpdxDocumentFileFunTest : StringSpec() {
             actualResult shouldBe expectedResult
         }
 
-        "mapDefinitionFiles() removes SPDX documents that do not describe a project" {
+        "Package dependencies are detected correctly, if no project is provided" {
+            val expectedResult = patchExpectedResult(
+                projectDir.parentFile.resolve("spdx-packages-expected-output.yml"),
+                url = vcsUrl,
+                urlProcessed = normalizeVcsUrl(vcsUrl),
+                revision = vcsRevision
+            )
+
+            val packageFileCurl = projectDir.resolve("package/libs/curl/package.spdx.yml")
+            val packageFileZlib = projectDir.resolve("package/libs/zlib/package.spdx.yml")
+
+            val definitionFiles = listOf(packageFileCurl, packageFileZlib)
+            // extracting projectResults to avoid depending on analyzer result specific items (e.g. dependency graph)
```

Comment:
Nit: This should be a full sentence, starting with a capital and ending with a dot.